### PR TITLE
i18n of login and register forms

### PR DESCRIPTION
**Pull request by martenson**
Feels like these two forms specifically could use it.

~~Needs help from a Spanish speaker.~~

Galaxy___martenson

## How to test the changes?
- [x] Instructions for manual testing are as follows:
  1. see login/register with your browser set to various locales

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: client/src/nls/es/locale.js [hunk 1]
```diff
@@ -627,8 +629,8 @@ define({
...
     "Public name": "Nombre público",
-    'Your public name is an identifier that will be used to generate addresses for information you share publicly. Public names must be at least three characters in length and contain only lower-case letters, numbers, and the "-" character.':
-        'Tu nombre público es un identificador que será utilizado para generar direcciones de información que compartas al público. Los nombres públicos deben contener al menos tres caracteres de longitud, solo letras minúsculas, números o el símbolo "-". ',
+    "Your public name is an identifier that will be used to generate addresses for information you share publicly. Public names must be at least three characters in length and contain only lower-case letters, numbers, dots, underscores, and dashes ('.', '_', '-').":
+        "Tu nombre público es un identificador que será utilizado para generar direcciones de información que compartas al público. Los nombres públicos deben contener al menos tres caracteres de longitud, solo letras minúsculas, números o el símbolo '-'. ",
```
Comment (martenson) [inline client/src/nls/es/locale.js:633]: this one is NOT translated, please help

**Comment by bgruening** [comment]
ping @gallardoalba and @davelopez

File: client/src/nls/es/locale.js (hunk 1 above)
Comment (davelopez) [reply client/src/nls/es/locale.js:633]:
```suggestion
        "Tu nombre público es un identificador que será utilizado para generar direcciones de información que compartas al público. Los nombres públicos deben contener al menos tres caracteres de longitud, solo letras minúsculas, números, puntos, guiones bajos o guiones ('.', '_', '-').",
```

**Comment by hexylena** [comment]
@bgruening maybe a better person to ping is @nomadscientist who has a team of people specifically working on translating technical spanish :) (Mostly since translation is a different activity than speaking/knowing the language.)

@nomadscientist can you pass this to the GTÑ team to check over? This file specifically contains the translations that are relevant. https://github.com/galaxyproject/galaxy/pull/14921/files#diff-d451c5c427138f20b9331f2a089ef2aefd94dae60c1127e514776429cf692187

**Comment by nomadscientist** [comment]
Hey! Actually at the moment, the person happens to be focusing on French
(but they are a translation studies MSc and are largely working on putting
in place stuff that will make is 100x easier to translate into any language
in the future, as I understand it) but have passed this into the GTÑ at
least

**Comment by davelopez** [comment]
No worries I'll push a commit with these missing translations :+1:

**Comment by martenson** [comment]
Thank you @davelopez!

**Pull request merged**